practice.md

```diff
@@ -49,6 +50,13 @@ You can then initialize a new 18F repository's license information with:
 
 It's also recommended to copy and paste [this paragraph for the end of your README](https://github.com/18F/open-source-policy/blob/master/README.md#public-domain) that sums up what's going on.
 
+### Accepting contributions from the public
+
+Any 18F project can accept open source contributions from the public. Contributors must agree to release their contributions into the international public domain.
```

Comment:
Updated with `can (and should!)`.

@benbalter Good point. I'm not totally sure. The [FSF FAQ](http://www.gnu.org/licenses/gpl-faq.html#GPLUSGovAdd) clears up a few points about US government and public domain code, but none quite cover the scenario of a government team writes a new WordPress plugin from scratch. I think in that situation, the "whole work" would still be unable to be GPL? And regardless, I'm not sure how it impacts what we can require of contributors not in our situation. Hrm.